trail/trail/urls.py

```diff
@@ -18,11 +18,31 @@
 from django.conf import settings
 from django.urls import path, re_path, include
 from django.views.generic import TemplateView
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+from rest_framework import permissions
+
+schema_view = get_schema_view(
+   openapi.Info(
+      title=settings.API_DOC_TITLE,
+      default_version=settings.API_DOC_DEFAULT_VER,
+      description=settings.API_DOC_DESCRIPTION,
+      terms_of_service="https://www.google.com/policies/terms/",
```

Comment:
Is this just a placeholder?

Reply:
Can you let me know which part you are talking about?
For the settings.py I will bring it up in the meeting and see if there are better solutions

Reply:
I think I got it from Dino. @DinoBektesevic correct me if I am wrong